ports/mimxrt/Makefile

```diff
@@ -277,7 +277,7 @@ SRC_QSTR += $(SRC_C) $(SHARED_SRC_C) $(GEN_PINS_SRC)
 
 CFLAGS += -g  # always include debug info in the ELF
 ifeq ($(DEBUG),1)
-CFLAGS += -Og
+CFLAGS += -O0
```

Comment:
I'm not sure why this is needed... we did already change `-O0` to `-Og` on some ports because O0 is too big to fit in flash.  And Og is intended for debugging, it keeps the code in a state where it's still straightforward to debug.

Reply:
Just a test leftover. With -Og, the debugger cannot match properly the code lines and has problems to show the values of local variables., probably due to optimization. I'll revert it.

Reply:
I reverted it. But for the MIMXRT port, there is sufficient flash for a firmware built with -O0.

Reply:
Thanks for reverting.

It's surprising the debugger cannot handle `-Og`...  if it's a problem we can change it, but it should be a separate commit and would be good to leave a comment in the Makefile as to why it's `-O0`, so it doesn't get changed again.

Reply:
In general, the debugger (ozone) can handle -Og. Only the match between the code line and the code is sometimes a little bit off, not allowing me so set breakpoints in the the C-Code view. In the disassembly view it works.